integration/duckdb/src/lance/duckdb/lance-extension.cc

```diff
@@ -14,47 +14,49 @@
 
 #define DUCKDB_EXTENSION_MAIN
 
+#include "lance-extension.h"
+
 #include <duckdb.hpp>
 
-#include "lance/duckdb/list_functions.h"
-#include "lance/duckdb/ml/functions.h"
-#include "lance/duckdb/vector_functions.h"
+#include "list_functions.h"
+#include "vector_functions.h"
```

Comment:
Do we plan to expose duckdb extension c++ code to others?
Should we maintain the namespace if so?